contracts/amm/libraries/IporSwapLogic.sol

```diff
@@ -159,4 +159,15 @@ library IporSwapLogic {
             revert(AmmErrors.UNSUPPORTED_SWAP_DURATION);
         }
     }
+
+    function getMaturity(AmmTypes.SwapDuration duration) internal pure returns (uint256) {
```

Comment:
it could be change in future to getTenor

Reply:
Yes in next pull requests related with refactor